matminer/featurizers/structure.py

```diff
@@ -1550,3 +1574,297 @@ def citations(self):
+            "min relative bond length"
+        ]
+        fl += [stat + " neighbor distance variation" for stat in self.stats]
+        fl.append("mean absolute deviation in relative cell size")
+        return fl
+
+    def citations(self):
+        return ["@article{Ward2017,"
+                "author = {Ward, Logan and Liu, Ruoqian "
+                "and Krishna, Amar and Hegde, Vinay I. "
+                "and Agrawal, Ankit and Choudhary, Alok "
+                "and Wolverton, Chris},"
+                "doi = {10.1103/PhysRevB.96.024104},"
+                "journal = {Physical Review B},"
+                "pages = {024104},"
+                "title = {{Including crystal structure attributes "
+                "in machine learning models of formation energies "
+                "via Voronoi tessellations}},"
+                "url = {http://link.aps.org/doi/10.1103/PhysRevB.96.014107},"
+                "volume = {96},year = {2017}}"]
+
+    def implementors(self):
+        return ['Logan Ward']
+
+
+class MaximumPackingEfficiency(BaseFeaturizer):
+    """Maximum possible packing efficiency of this structure
+
+    Uses a Voronoi tessellation to determine the largest radius each atom
+    can have before any atoms touches any one of their neighbors. Given the
+    maximum radius size, this class computes the maximum packing efficiency
+    of the structure as a feature.
+
+    Features:
+        max packing efficiency - Maximum possible packing efficiency
+    """
+
+    def featurize(self, strc):
+        # Get the Voronoi tessellation of each site
+        voro = VoronoiNN()
+        nns = [voro.get_voronoi_polyhedra(strc, i) for i in range(len(strc))]
+
+        # Compute the radius of largest possible atom for each site
+        #  The largest radius is equal to the distance from the center of the
+        #   cell to the closest Voronoi face
+        max_r = [min(x['face_dist'] for x in nn.values()) for nn in nns]
+
+        # Compute the packing efficiency
+        return 4. / 3. * np.pi * np.power(max_r, 3).sum() / strc.volume
+
+    def feature_labels(self):
+        return ['max packing efficiency']
+
+    def citations(self):
+        return ["@article{Ward2017,"
+                "author = {Ward, Logan and Liu, Ruoqian "
+                "and Krishna, Amar and Hegde, Vinay I. "
+                "and Agrawal, Ankit and Choudhary, Alok "
+                "and Wolverton, Chris},"
+                "doi = {10.1103/PhysRevB.96.024104},"
+                "journal = {Physical Review B},"
+                "pages = {024104},"
+                "title = {{Including crystal structure attributes "
+                "in machine learning models of formation energies "
+                "via Voronoi tessellations}},"
+                "url = {http://link.aps.org/doi/10.1103/PhysRevB.96.014107},"
+                "volume = {96},year = {2017}}"]
+
+    def implementors(self):
+        return ['Logan Ward']
+
+
+class ChemicalOrdering(BaseFeaturizer):
```

Comment:
Can you clarify (for my knowledge) the difference between this and the ChemicalSRO site descriptor?

Reply:
It appears that ChemicalSRO breaks down the degree of ordering to each particular element. ChemicalOrder is - in some ways - a coarse average over all the ChemicalSROs for each type of element in the structure.

Does that help clarify? 

I'm currently polishing the documentation of each class to make the difference clearer.

Reply:
ok